### ISSUE TITLE
Gallery view support

### ISSUE THREAD

**Issue by luminous8** [dogancelik/twdl#3]
Hey, awesome package!

Do you plan to support when there's mulitple images in a tweet (like this one https://twitter.com/EdChirac/status/1297050053496967169)?

Also is there a way to use it like a nodejs module?

Thanks

**Comment by dogancelik** [comment]
Oh hey, thanks for asking.

>Do you plan to support when there's mulitple images in a tweet

I coded so it only fetches the images of the tweet author, not the images of the quoted tweet.

**Edit:** If you want to download the quoted images too, you can use the `-u` flag. Check `twdl --help` for other flags.

```
twdl -u https://twitter.com/EdChirac/status/1297050053496967169
```

> Also is there a way to use it like a nodejs module?

Yes, you can check [the test directory](https://github.com/dogancelik/twdl/blob/master/test/index.js) for basic examples. I shall add examples to the wiki soon.

**Comment by luminous8** [comment]
oh yes, that's perfect (I didn't notice it was a quoted tweet).

Awesome for the nodejs module, would it be possible to have a function that return the image urls?

To use like this:
```
const twdl = require('twdl');
const urls = await twdl.getImages(['https://twitter.com/NVIDIAGeForce/status/1258069454312755201'])
```

Thanks

**Comment by dogancelik** [comment]
I published `v1.7.1`

You can get URLs with a custom function:

```js
const twdl = require('twdl');

var mediaUrls = [];

function downloadUrl(mediaUrl, tweetUrl, mediaData, options) {
	mediaUrls.push(mediaUrl);
	return mediaUrl;
}

var results = await twdl.downloadUrls(tweetUrls, { downloadUrlFn: downloadUrl });

console.log('Media URLs', mediaUrls);
console.log('Results', results);
```

I may change the property name `downloadUrlFn` in the future so be sure to check the wiki for up-to-date examples.
